schematron/v2.2.0/exemplary_files/L200_Audit.xml

```diff
@@ -142,60 +138,401 @@
-            <!-- note that LampType must be defined -->
+          <auc:LightingSystem ID="LightingSystem-A">
             <auc:LampType>
               <auc:CompactFluorescent>
-                <!-- LampLabel is recommended, not required -->
-                <auc:LampLabel>2D</auc:LampLabel>
+                <auc:LampLabel>Spiral</auc:LampLabel>
               </auc:CompactFluorescent>
             </auc:LampType>
+            <auc:BallastType>Standard Electronic</auc:BallastType>
+            <auc:DimmingCapability>
+              <auc:MinimumDimmingLightFraction>.2</auc:MinimumDimmingLightFraction>
+            </auc:DimmingCapability>
+            <auc:PercentPremisesServed>100</auc:PercentPremisesServed>
+            <auc:InstalledPower>123</auc:InstalledPower>
+            <auc:LampPower>123</auc:LampPower>
+            <auc:NumberOfLampsPerBallast>1</auc:NumberOfLampsPerBallast>
+            <auc:NumberOfBallastsPerLuminaire>1</auc:NumberOfBallastsPerLuminaire>
+            <auc:NumberOfLuminaires>1</auc:NumberOfLuminaires>
+            <auc:OutsideLighting>false</auc:OutsideLighting>
+            <auc:Controls>
+              <auc:Control>
+                <auc:Daylighting>
+                  <auc:ControlSystemType>
+                    <auc:Digital/>
+                  </auc:ControlSystemType>
+                  <auc:ControlSensor>Photocell</auc:ControlSensor>
+                  <auc:ControlStrategy>Continuous</auc:ControlStrategy>
+                </auc:Daylighting>
+              </auc:Control>
+            </auc:Controls>
+            <auc:LightingAutomationSystem>false</auc:LightingAutomationSystem>
             <auc:LinkedPremises>
               <auc:Section>
-                <auc:LinkedSectionID IDref="Section-Space-A"></auc:LinkedSectionID>
+                <auc:LinkedSectionID IDref="Section-Space-A">
+                  <auc:LinkedScheduleIDs>
+                    <auc:LinkedScheduleID IDref="LightingSchedule-Original"/>
+                  </auc:LinkedScheduleIDs>
+                </auc:LinkedSectionID>
+                <auc:LinkedSectionID IDref="Section-Space-B">
+                  <auc:LinkedScheduleIDs>
+                    <auc:LinkedScheduleID IDref="LightingSchedule-Original"/>
+                  </auc:LinkedScheduleIDs>
+                </auc:LinkedSectionID>
               </auc:Section>
             </auc:LinkedPremises>
           </auc:LightingSystem>
         </auc:LightingSystems>
-
+        <auc:PumpSystems>
+          <auc:PumpSystem ID="PumpSystem-B">
+            <auc:PumpControlType>Constant Volume</auc:PumpControlType>
+            <auc:LinkedSystemIDs>
+              <auc:LinkedSystemID IDref="CoolingPlant-A"/>
+            </auc:LinkedSystemIDs>
+          </auc:PumpSystem>
+        </auc:PumpSystems>
+        <auc:FanSystems>
+          <auc:FanSystem ID="FanSystem-A">
+            <auc:FanControlType>Stepped</auc:FanControlType>
+            <auc:LinkedSystemIDs>
```

Comment:
For fans, I think we should also require:
- FanEfficiency
- FanSize
- InstalledFlowRate [WARNING]

Reply:
Can we also add a PR to the schema to:
- deprecate FanSize
- Add `FanMaximumFlowRate` (to mirror the pump).  This is replacing FanSize.  Assuming that gets approved, change the above to require `FanMaximumFlowRate` instead of `FanSize`

Reply:
Also, to mirror pump, change `InstalledFlowRate` -> `FanInstalledFlowRate`

Reply:
First part addressed here: ca9a4ca2df94a1257842f30091c0ec77d26ce3d0

I think we should punt on adding these new elements for now and just get this PR merged, then tweak as we need to later.

Reply:
kk